build/yaml/runUnitTests-CI.yml

```diff
@@ -0,0 +1,27 @@
+steps:
```

Comment:
Can we combine this template with functionalTestsBuild-CI.yml? It seems that functionalTestsBuild-CI.yml already does UseDotNet@2 and also does a build and we are doing that here again. Let me know what you think but I believe that at this point we can have them combined.
If we decide to separate them, then we should look into taking the build output and run the tests on that rather than recompiling again.

Reply:
Feedback applied. Thanks!